apps/docs/src/app/platform/component-docs/platform-table/platform-table-examples/platform-table-navigatable-row-example.component.ts

```diff
@@ -0,0 +1,176 @@
+import { Component, ViewChild, AfterViewInit, ChangeDetectionStrategy } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+import { FdDate } from '@fundamental-ngx/core';
+import {
+    TableDataSource,
+    TableDataProvider,
+    TableState,
+    TableRowSelectionChangeEvent,
+    TableComponent
+} from '@fundamental-ngx/platform';
+
+@Component({
+    selector: 'fdp-platform-table-navigatable-row-example',
+    templateUrl: './platform-table-navigatable-row-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformTableNavigatableRowExampleComponent implements AfterViewInit {
+
+    source: TableDataSource<ExampleItem>;
+
+    @ViewChild(TableComponent)
+    table: TableComponent;
+
+    constructor() {
+        this.source = new TableDataSource(new TableDataProviderExample());
+    }
+
+    ngAfterViewInit(): void {
+        this.setRowUnnavigatable(0);
+        this.setRowUnnavigatable(1);
+        this.setRowUnnavigatable(2);
+    }
+
+    onRowSelectionChange(event: TableRowSelectionChangeEvent<ExampleItem>): void {
+        console.log(event);
+    }
+
+    private setRowUnnavigatable(index: number): void {
+        this.table.tableRows.toArray()[index + 1].navigatable = false;
```

Comment:
I would simplify this for one-liner for the customer so he doesn't need to write 2 lines of code, keeping in mind that one index is +1 than the other

Reply:
The first line is referring to ```tableRows``` and the second line is referring to ```_tableRows```. So perhaps we can't simplify in this case?

Reply:
You have access to TableComponent, so you can add a unified method for changing navigatable property for the desired row. This actually is more convenient and clearer way. So in basic you would have:
```typescript
setNavigatableRowState(rowIndex, state) {
    this.tableRows.toArray()[rowIndex + 1].navigatable = state;
    this._tableRows[rowIndex].navigatable = state;
}
```
in `TableComponent` class.

And call it from example component in such manner:
```typescript
this.table.setNavigatableRowState(index, false);
```

Reply:
Done. Thanks a lot for clarifying!